tests/mswServer.js

```diff
@@ -0,0 +1,3 @@
+import { setupServer } from 'msw/node';
+
+export const mswServer = setupServer();
```

Comment:
I think I'd replace this for a default export, feels more natural to me when importing, since this is a single setup 🤔 

```js
const mswServer = setupServer();

export default mswServer;
```

Reply:
Done